Gears/VehicleData.hpp

```diff
@@ -72,6 +72,16 @@ struct VehicleGearboxStates {
     float EngineLoad = 0.0f;
     float UpshiftLoad = 0.0f;
     float DownshiftLoad = 0.0f;
+
+    // TCU stuff
```

Comment:
You might want to use a new struct for TCU-specific stuff, since you also have your own update thing.